gradlew

```diff
@@ -69,20 +69,19 @@ app_path=$0
 
 # Need this for daisy-chained symlinks.
 while
-    APP_HOME=${app_path%"${app_path##*/}"}  # leaves a trailing /; empty if no leading path
-    [ -h "$app_path" ]
+  APP_HOME=${app_path%"${app_path##*/}"} # leaves a trailing /; empty if no leading path
+  [ -h "$app_path" ]
 do
-    ls=$( ls -ld "$app_path" )
-    link=${ls#*' -> '}
-    case $link in             #(
-      /*)   app_path=$link ;; #(
-      *)    app_path=$APP_HOME$link ;;
-    esac
+  ls=$(ls -ld "$app_path")
+  link=${ls#*' -> '}
+  case $link in         #(
+  /*) app_path=$link ;; #(
+  *) app_path=$APP_HOME$link ;;
+  esac
 done
 
-APP_HOME=$( cd "${APP_HOME:-./}" && pwd -P ) || exit
+APP_HOME=$(cd "${APP_HOME:-./}" && pwd -P) || exit
 
-APP_NAME="Gradle"
```

Comment:
This constant wasn't used in the whole project.

Reply:
Yhea because its a Autogenerated File if you use `gradle init`

Reply:
Ok, I restore this line of code in the commit: [bd3d1f4](https://github.com/airvzxf/Bitburner-Plugin/commit/bd3d1f415769f641d6596cda2757bad47b2eb892).